docs/INSTALL.md

```diff
@@ -20,13 +21,13 @@ On **Debian**:
 
 ```bash
 apt update
-apt install gettext postgresql postgresql-contrib libpq-dev git
+apt install gettext postgresql postgresql-contrib libpq-dev git curl gcc make openssl libssl-dev 
 ```
 
 On **Fedora**, **CentOS** or **RHEL**:
 
 ```bash
-dnf install postgresql-server postgresql-contrib mariadb-devel libsq3-devel libpqxx libpqxx-devel
+dnf install postgresql-server postgresql-contrib mariadb-devel libsq3-devel libpqxx libpqxx-devel git curl gcc make openssl libssl-dev 
 # TODO: GetText + Git install
```

Comment:
Could you remove the "Git install" here since you added it?

Reply:
Yes, I remove it, sorry